health/health.d/net.conf

```diff
@@ -1,6 +1,24 @@
 
 # you can disable an alarm notification by setting the 'to' line to: silent
 
+# -----------------------------------------------------------------------------
+# net traffic overflow
+
+ template: 10s_net_traffic_overflow
+       on: net.net
+       os: linux
+    hosts: *
+ families: *
+   lookup: average -10s unaligned absolute of sent,received
```

Comment:
I think we need 2 alarms, one for `received` and another for `sent`.
The way this is now, `$this` will be average of the **sum** of `sent` and `average`.